### ISSUE TITLE
Issue requesting tv shows

### ISSUE THREAD

**Issue by Samus21** [Ombi-app/Ombi#5009]
### Summary

Ive been having an issue recently where if i use the app and request all seasons or try to select multiple episodes in a season the box never pops up to select folder, quality, ect. I did a fresh install and set ombi up again. It worked without issues for about 2 days and now it wont let me request with the app again. I can request movies fine. all my setting for sonaar match and it connects fine when i test the connection. I also have it show me Indian movies on the app when i have selected US as my country.

![IMG_9186](https://github.com/Ombi-app/Ombi/assets/143613370/bb0f709f-5fd6-4a60-a08f-edd7b4f42bb3)
![IMG_9188](https://github.com/Ombi-app/Ombi/assets/143613370/a7a88f9a-d47e-4c10-a695-1fa359163407)
![IMG_9187](https://github.com/Ombi-app/Ombi/assets/143613370/fc3b85bf-d37d-427a-ac24-b6a6eaaa49c2)

### Ombi Version

 4.43.5

### What platform(s) does this occur on?

Windows

### What database are you using?

SQLite (Default)

**Comment by Samus21** [comment]
fail: Ombi.ErrorHandlingMiddleware[0]
      Something bad happened, ErrorMiddleware caught this
      System.NullReferenceException: Object reference not set to an instance of an object.
         at Ombi.Controllers.V1.External.PlexController.GetFriends() in /home/runner/work/Ombi/Ombi/src/Ombi/Controllers/V1/External/PlexController.cs:line 260
         at lambda_method3728(Closure , Object )
         at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
      --- End of stack trace from previous location ---
         at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
      --- End of stack trace from previous location ---
         at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
         at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
         at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
         at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
         at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
         at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
         at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
         at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
         at Ombi.ApiKeyMiddlewear.Invoke(HttpContext context) in /home/runner/work/Ombi/Ombi/src/Ombi/Middleware/ApiKeyMiddlewear.cs:line 58
         at Ombi.ErrorHandlingMiddleware.Invoke(HttpContext context) in /home/runner/work/Ombi/Ombi/src/Ombi/Middleware/ErrorHandlingMiddlewear.cs:line 30

**Comment by bejaranoo** [comment]
Having a similar issue, I as an admin can select path, and profile hence submit a sonarr request no problem.
But when a normal user tries to request neither of those options are shown, I guess the correct solution would be to automatically use the default path and profile saved on sonarr config, but instead it seems to be sending the request with those fields blank:
```
fail: Ombi.Api.Api[1000]

      StatusCode: BadRequest, Reason: Bad Request, RequestUri: http://sonarr:8989/api/v3/series/

fail: Ombi.Api.Api[1000]

      StatusCode: BadRequest, Reason: Bad Request, RequestUri: http://sonarr:8989/api/v3/series/

fail: Ombi.Core.Senders.TvSender[5001]

      Exception thrown when attempting to send series over to Sonarr

      System.Exception: Invalid Path,Invalid Path

         at Ombi.Core.Senders.TvSender.SendToSonarr(ChildRequests model, SonarrSettings s) in /home/runner/work/Ombi/Ombi/src/Ombi.Core/Senders/TvSender.cs:line 318

fail: Ombi.Core.Senders.TvSender[0]

      Exception thrown when sending a movie to DVR app, added to the request queue

      System.Exception: Invalid Path,Invalid Path

         at Ombi.Core.Senders.TvSender.SendToSonarr(ChildRequests model, SonarrSettings s) in /home/runner/work/Ombi/Ombi/src/Ombi.Core/Senders/TvSender.cs:line 318

         at Ombi.Core.Senders.TvSender.Send(ChildRequests model) in /home/runner/work/Ombi/Ombi/src/Ombi.Core/Senders/TvSender.cs:line 60
```

**Comment by tidusjar** [comment]
> I guess the correct solution would be to automatically use the default path and profile saved on sonarr config

This does happen, are you sure that you have a correct path set in the Ombi sonarr settings?

One thing that can happen is that if you delete and recreate a path in Sonarr, Ombi does not know about this and will use the exising now invalid path

**Comment by Samus21** [comment]
Since I made this post I reinstalled my entire OS. The paths have not changed for storage. I’m using Microsoft storage pool and none of that was messed with when the os was reinstalled. I’ve seen a few others recently reporting the same issue on Reddit and the discord. Seems like it’s an iOS issue. I have no way to confirm if androids have the issue. I can request all seasons on my pc and if I don’t use the app and log in directly to the website it will request all seasons. So it’s something to do with the app and possibly iOS
